fdbserver/BackupWorker.actor.cpp

```diff
@@ -425,7 +425,7 @@ struct BackupData {
 
 	ACTOR static Future<Version> _getMinKnownCommittedVersion(BackupData* self) {
 		loop {
-			GetReadVersionRequest request(1, TransactionPriority::DEFAULT,
+			GetReadVersionRequest request(0, TransactionPriority::DEFAULT,
 			                                     GetReadVersionRequest::FLAG_USE_MIN_KNOWN_COMMITTED_VERSION);
 			choose {
 				when(wait(self->cx->onMasterProxiesChanged())) {}
```

Comment:
The reason GRV requests need to be balanced by number of requests is because of the interaction with ratekeeper. For requests like this one that do not take any ratekeeper budget, it is better to send the request to the least utilized proxy.